schema.graphql

```diff
@@ -564,3 +564,12 @@ type Liquidation @entity {
   fCashMaturities: [BigInt!]
   fCashNotionalTransfer: [BigInt!]
 }
+
+# Data accumulated and condensed into day stats for lending and borrowing
+type LendBorrowDayData @entity {
+  id: ID!
+  date: Int!
+  volumeNetUnderlyingCash: BigInt!
```

Comment:
It's probably inaccurate to call this "NetUnderlyingCash" because you are taking the `abs()` and it is actually the total volume in either Borrow or Lend. Maybe this should just be `totalVolumeUnderlyingCash`?